### PR TITLE
Add cmd to retrieve the current-csv from the manifests directory

### PR DESCRIPTION
**Pull request by b1zzu**
**What**

The cmd will retrieve the current CSV from the manifests directory containing multiple CSVs

**Verify**

1. Download and extract the manifests directory

```
export SRC_IMAGE='registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata@sha256:8355860dbc1052884138612fe1b12a149ceee58440831c041eb3fd45bb38a52e'
mkdir /tmp/mydir
./delorean ews extract-manifests --src-image ${SRC_IMAGE} --extract-dir /tmp/mydir
```
 
2. Retrieve the current CSV from `/tmp/mydir`

```
delorean ews current-csv --directory /tmp/mydir --output /tm/tmpcurrentcsv.json
```

### CODE REVIEW

File: cmd/currentCSV.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		Use:   "current-csv",
+		Short: "Retrieve the current CSV from the manifests directory and write it in JSON format to the output file",
+		Run: func(cmd *cobra.Command, args []string) {
+			csv, file, err := utils.GetCurrentCSV(flags.directory)
+			if err != nil {
+				handleError(err)
+			}
+
+			fmt.Printf("Write current CSV %s to %s\n", file, flags.output)
+			bytes, err := json.Marshal(csv)
```
Comment (mikenairn) [inline cmd/currentCSV.go:31]: Any reason we need to write this as json instead of yaml?

File: cmd/currentCSV.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+			_, err = write.Write(bytes)
+			if err != nil {
+				handleError(err)
+			}
+
+			err = write.Close()
+			if err != nil {
+				handleError(err)
+			}
```
Comment (mikenairn) [inline cmd/currentCSV.go:50]: Probably better to add a helper like the one here for the yaml version https://github.com/integr8ly/delorean/blob/master/pkg/utils/io.go#L41

**Comment by mikenairn** [comment]
@b1zzu Does cobra have any support for just returning json/yaml as output from the commands?

File: cmd/currentCSV.go (hunk 1 above)
Comment (b1zzu) [reply cmd/currentCSV.go:31]: It's easy to parse JSON, for example, Jenkins has the built-in function, and cmd like `jq` is available on all distro on opposite to `yq`

But I found out there is a built-in function to read yaml too in Jenkins, so I could also write to yaml

**Comment by b1zzu** [comment]
@mikenairn should be ready now, I've added the util method to write to JSON

**Pull request merged**